WalletWasabi/BitcoinCore/Rpc/Models/TransactionBlockInfo.cs

```diff
@@ -0,0 +1,23 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.BitcoinCore.Rpc.Models
+{
+	public class TransactionBlockInfo
+	{
+		public TransactionBlockInfo(uint256 blockHash, DateTimeOffset? blockTime, uint? blockIndex)
```

Comment:
Currently, `blockIndex` is assigned always non-null value. Maybe it's not needed?

Similarly for `blockTime`.

Reply:
Done!